src/stan-users-guide/efficiency-tuning.Rmd

```diff
@@ -189,7 +189,7 @@ support for values in the hundreds or even thousands, whereas the
 posterior may be concentrated near zero).  In such cases, replacing
 the fixed prior with a hierarchical prior such as `normal(mu,
   sigma)`, where `mu` and `sigma` are new parameters, with
-their own hyperpriors.
+their own hyperpriors can be beneficial.
```

Comment:
The new version isn't grammatical.  The fix should probably be to remove the comma before "with".

Reply:
You're right, see the updated version.